examples/damsvalley.jl

```diff
@@ -92,7 +92,7 @@ const MAX_ITER = 40
 """Build probability distribution at each timestep.
 Return a Vector{NoiseLaw}"""
 function generate_probability_laws()
-    laws = Vector{NoiseLaw}(N_STAGES-1)
+    laws = Vector{NoiseLaw}(undef,N_STAGES-1)
```

Comment:
Oups, I think I forget to flag this `undef` in my previous review.

Reply:
fixed